apollo-router/src/plugins/README.md

```diff
@@ -0,0 +1,222 @@
+# Apollo Router Plugin System
+
+## Overview
+
+Apollo Router supports a modular plugin system that allows you to customize and extend its behavior at various stages of request processing. Plugins are compiled into the router and configured via YAML configuration. Each plugin can hook into different phases of the request lifecycle, such as authentication, authorization, caching, telemetry, and more.
```

Comment:
> Each plugin can hook into different phases of the request lifecycle, such as authentication, authorization, caching, telemetry, and more.

this doesn't really make sense. those are examples of plugins, not of request lifecycle phases (router, supergraph, execution, subgraph...)